### PR TITLE
dev-db/redis: bump to 4.0.14/5.0.4

### PR DESCRIPTION
**Pull request by hydrapolic**
Thanks.

### CODE REVIEW

File: dev-db/redis/redis-5.0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline dev-db/redis/redis-5.0.4.ebuild]: why not 7?

Comment (hydrapolic) [reply dev-db/redis/redis-5.0.4.ebuild]: Bumped to eapi 7 ;)

File: dev-db/redis/redis-4.0.14.ebuild [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+# changes.
+RDEPEND="
+	luajit? ( dev-lang/luajit:2 )
+	!luajit? ( || ( dev-lang/lua:5.1 =dev-lang/lua-5.1*:0 ) )
+	tcmalloc? ( dev-util/google-perftools )
+	jemalloc? ( >=dev-libs/jemalloc-3.2 )"
+
+DEPEND="
+	${RDEPEND}
+	virtual/pkgconfig
```
Comment (Zlogene) [inline dev-db/redis/redis-4.0.14.ebuild]: well, I actually expected you to move these to `BDEPEND`

File: dev-db/redis/redis-4.0.14.ebuild [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+	>=sys-devel/autoconf-2.63
+	test? ( dev-lang/tcl:0= )"
```
Comment (Zlogene) [inline dev-db/redis/redis-4.0.14.ebuild]: and this too

File: dev-db/redis/redis-4.0.14.ebuild [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+RDEPEND="
+	luajit? ( dev-lang/luajit:2 )
+	!luajit? ( || ( dev-lang/lua:5.1 =dev-lang/lua-5.1*:0 ) )
+	tcmalloc? ( dev-util/google-perftools )
+	jemalloc? ( >=dev-libs/jemalloc-3.2 )"
+
+DEPEND="
+	${RDEPEND}
+	virtual/pkgconfig
+	>=sys-devel/autoconf-2.63
```
Comment (Zlogene) [inline dev-db/redis/redis-4.0.14.ebuild]: you probably wanna use `WANT_AUTOCONF` variable, not the dependency string

File: dev-db/redis/redis-5.0.4.ebuild (hunk 1 above)
Comment (Zlogene) [reply dev-db/redis/redis-5.0.4.ebuild]: but this was not enough :)

File: dev-db/redis/redis-4.0.14.ebuild (hunk 3 above)
Comment (hydrapolic) [reply dev-db/redis/redis-4.0.14.ebuild]: Not sure how to handle this. By checking sqlite it seems it's in BDEPEND and DEPEND too.

**Comment by hydrapolic** [comment]
@Zlogene, pushed an update, unsure about how to handle tcl :-/

**Comment by robbat2** [comment]
Why did you drop the slotted Lua support in 5.0.x?

**Comment by robbat2** [comment]
Merged; thank you.

**Comment by hydrapolic** [comment]
> Why did you drop the slotted Lua support in 5.0.x?

This PR was created before the LUA adjustments were made to 5.0.x, sorry I forgot to update the PR.

**Comment by hydrapolic** [comment]
> Merged; thank you.

Thank you @robbat2

**Pull request closed without merging**